pkg/openstack/loadbalancer_test.go

```diff
@@ -1,13 +1,15 @@
 package openstack
 
 import (
+	"fmt"
 	"sort"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
-
 	"github.com/gophercloud/gophercloud/openstack/loadbalancer/v2/listeners"
 	"github.com/gophercloud/gophercloud/openstack/networking/v2/extensions/security/rules"
+	"github.com/stretchr/testify/assert"
```

Comment:
Feels like we tried to keep this one separated for a reason, I guess because it's a testing-only import? No big deal here though.